plugins/inputs/netflow/README.md

```diff
@@ -0,0 +1,101 @@
+# Netflow Input Plugin
+
+The `netflow` plugin acts as a collector for Netflow v5, Netflow v9 and IPFIX
```

Comment:
Is IPFIX v10? After reading the wikipedia article I wasn't sure.

Reply:
Yes it is. However, IPFIX is the more common name, so I put that one in. It's also used in the spec...